server/routerlicious/packages/services/src/tenant.ts

```diff
@@ -260,7 +262,25 @@ export class TenantManager implements core.ITenantManager, core.ITenantConfigMan
 			undefined /* refreshTokenIfNeeded */,
 			logHttpMetrics,
 		);
-		await restWrapper.post(`/api/tenants/${encodeURIComponent(tenantId)}/validate`, { token });
+		try {
+			await restWrapper.post(`/api/tenants/${encodeURIComponent(tenantId)}/validate`, {
+				token,
+			});
+		} catch (error: unknown) {
+			if (isNetworkError(error)) {
+				if (error.code === 401 || error.code === 403) {
```

Comment:
There are several other places, before contacting Riddler, that we attempt to check various token claims to make sure the token is valid. However, I think the most relevant spots for this are `validateTokenClaims` and `validateTokenClaimsExpiration` in `services-client/src/auth.ts`.

I think,
1. if we're trying to log as many 401/403 JTI's as possible, we should also log up where those are called.
2. if we're only concerned about Riddler's responses, then we should put this telemetry in Riddler itself, rather than here. Otherwise, you should also add this same exact logic into Historian's `RiddlerService.verifyToken`